docs/src/main/tut/streaming.md

```diff
@@ -127,7 +124,6 @@ object service {
      * @return Stream of server responses.
      */
     @rpc(Protobuf)
-    @stream[ResponseStreaming.type]
```

Comment:
What is happened to the `@stream` annotation? Is it no longer supported or no longer needed?